medicine/run.py

```diff
@@ -37,14 +36,15 @@ def run_medicine(
     learning_rate: float = 0.0005,
     epsilon: float = 1e-3,
     plot_figures: bool = True,
+    return_motion: bool = True,
```

Comment:
maybe we could remove this and always return the motion.
I wanted to make it False by default.

Reply:
sure, then we could return the 3 arrays

Reply:
Great!  I made some suggested edits to return the trainer and 3 arrays (I'll make sure this is backwards compatible with the paper code).